client-course-schedulizer/src/utilities/interfaces/dataInterfaces/scheduleInterfaces.ts

```diff
@@ -21,47 +21,46 @@ export interface Meeting {
 export type Prefix = string;
 
 export interface Course {
-  department: string;
-  facultyHours: number;
+  department?: string;
   name: string;
   number: string;
   prefixes: Prefix[];
   sections: Section[];
-  studentHours: number;
 }
 
 export type Instructor = string;
 
 export interface Section {
   anticipatedSize?: number;
-  comments: string;
+  comments?: string;
   // Number of students enrolled in this section 10 days into the course
   day10Used?: number;
   // Like 2/3/2020
-  endDate: string;
+  endDate?: string;
   // Overrides Course value
```

Comment:
remove this comment since it doesn't override the course value anymore (facultyHours isn't on the course datatype anymore).